backend/spec/features/admin/configuration/countries_spec.rb

```diff
@@ -9,7 +9,7 @@ module Spree
       click_link "New Country"
 
       fill_in "Name", with: "Brazil"
-      fill_in "Iso Name", with: "BRL"
+      fill_in "ISO Name", with: "BRL"
```

Comment:
To avoid errors like this in the future we could use the transition in the test. 

```
fill_in Spree::Country.human_attribute_name(:iso_name), with: "BRL"
```